registry.json

```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "ticker": "LATIN",
+    "poolId": "e8e00995c7f3820fedf01e75a443c56e20f96ddcf9168b662ce2fdcd",
+    "social": {
+      "website": "cardanolatino.com",
+      "discord": "discord.gg/bJVVtwsd9Y",
+      "reddit": "u/CardanoLatino",
+      "instagram": "cardanolatinousa",
+      "facebook": "https://www.facebook.com/CardanoLatinoUS/",
+      "youtube": "UCOJ_TtwIG7F-mzadfkbmfGg",
+      "twitter": "CardanoLatino"
+    },
```

Comment:
You need to add telegram information - this must be for an individual and not channel or group.

Also please add the github and operatorName keys in, even if you are leaving blank